Automattic/wp-calypso
# Update /plans features taglines

Updated taglines for Blogger, Personal, Premium, Business, and
eCommerce.

Repeated Business features on the eCommerce plan full feature view and
new eComm-specific features at the bottom like other plans.

Updated Blogger domain feature + tooltip.

Added more descriptive tooltips for new eComm features.

**Testing instructions:**

Go to WordPress.com/plans to see the new tabbed experience and the applied copy changes.

In addition, go to My Sites > Add new site and on the Plans page of the signup process, you should see updated copy and tooltips for all the features.
